src/lib/Tests/FieldType/FieldTypeFormMapperDispatcherTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+
+namespace EzSystems\EzPlatformContentForms\Tests\FieldType;
+
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\Repository\Values\ContentType\FieldDefinition;
+use EzSystems\EzPlatformContentForms\Data\Content\FieldData;
+use EzSystems\EzPlatformContentForms\FieldType\FieldTypeFormMapperDispatcher;
+use EzSystems\EzPlatformContentForms\FieldType\FieldTypeFormMapperDispatcherInterface;
+use EzSystems\EzPlatformContentForms\FieldType\FieldValueFormMapperInterface;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\FormInterface;
+
+class FieldTypeFormMapperDispatcherTest extends TestCase
+{
+    /**
```

Comment:
FQCN and single line, dont know if this is fixer fault but those should be checked?